airflow/cli/commands/scheduler_command.py

```diff
@@ -29,17 +29,21 @@
 from airflow.utils.cli import process_subdir, setup_locations, setup_logging, sigint_handler, sigquit_handler
 
 
+def create_scheduler_job(args):
```

Comment:
```suggestion
def _create_scheduler_job(args):
```

nit: Probably just to suggest that it is internal function